### PR TITLE
Stops using the rad insulation component to provide default rad insulation features in order to save a bit of memory

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
Also contains a typo fix

:cl: Naksu
code: default radiation insulation is now handled by atom vars rather than a component, components can still be used.
/:cl:

This saves around 100MB of roundstart memory.

### CODE REVIEW

File: code/datums/components/mirage_border.dm [hunk 1]
```diff
@@ -4,7 +4,7 @@
 /datum/component/mirage_border/Initialize(turf/target, direction, range=world.view)
 	if(!isturf(parent))
 		return COMPONENT_INCOMPATIBLE
-	if(!target || !istype(target) || !direction)
```
Comment (deathride58) [inline code/datums/components/mirage_border.dm]: undocumented change?

Comment (vuonojenmustaturska) [reply code/datums/components/mirage_border.dm]: `Also contains a typo fix and removes an impossible if test from mirage_border`

Comment (vuonojenmustaturska) [reply code/datums/components/mirage_border.dm]: what was i thinking yesterday

Comment (vuonojenmustaturska) [reply code/datums/components/mirage_border.dm]: oh whatever the code actually works but i did not think this through

File: code/game/atoms.dm [hunk 2]
```diff
@@ -36,6 +36,9 @@
 
 	var/datum/component/orbiter/orbiters
 
+	var/rad_flags = NONE // Will move to flags_1 when i can be arsed to
```
Comment (Cyberboss) [inline code/game/atoms.dm]: reee

File: code/game/objects/structures/window.dm [hunk 3]
```diff
@@ -25,7 +25,8 @@
 	var/real_explosion_block	//ignore this, just use explosion_block
 	var/breaksound = "shatter"
 	var/hitsound = 'sound/effects/Glasshit.ogg'
-	var/rad_insulation = RAD_VERY_LIGHT_INSULATION
+	rad_insulation = RAD_VERY_LIGHT_INSULATION
+	rad_flags = RAD_PROTECT_CONTENTS
```
Comment (Cyberboss) [inline code/game/objects/structures/window.dm]: Above new vars

File: code/modules/power/singularity/collector.dm [hunk 4]
```diff
@@ -26,14 +26,11 @@
 
 	var/bitcoinproduction_drain = 0.15
 	var/bitcoinmining = FALSE
+	rad_insulation = RAD_EXTREME_INSULATION
```
Comment (Cyberboss) [inline code/modules/power/singularity/collector.dm]: ^

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Pull request merged**